search-commons/src/test/java/no/unit/nva/search/IndexDocumentWrapperLinkedDataTest.java

```diff
@@ -0,0 +1,94 @@
+package no.unit.nva.search;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.contexttypes.Journal;
+import no.unit.nva.publication.PublicationGenerator;
+import no.unit.nva.utils.IndexDocumentWrapperLinkedData;
+import no.unit.nva.utils.UriRetriever;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Path;
+
+import static nva.commons.core.JsonUtils.objectMapper;
+import static nva.commons.core.ioutils.IoUtils.stringFromResources;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+class IndexDocumentWrapperLinkedDataTest {
+
+    public static final String TYPE_JSON_PTR = "/entityDescription/reference/publicationContext/type";
+    public static final String NAME_JSON_PTR = "/entityDescription/reference/publicationContext/name";
+    public static final String SERIES_ID_JSON_PTR = "/entityDescription/reference/publicationContext/id";
+    public static final String JOURNAL_NAME = "name of the journal";
+    public static final String CONTEXT_OBJECT_TYPE = "Journal";
+    public static final String SAMPLE_JSON_FILENAME = "framed-json/publication_channel_sample.json";
+    public static final String FIELD_NAME = "name";
+    public static final String EXAMPLE_CHANNEL_URI =
+            "https://api.dev.nva.aws.unit.no/publication-channels/journal/495273/2020";
+
+    @Test
+    public void toFramedJsonLdReturnsJsonWithValidReferenceData() throws Exception {
+
+        // https://github.com/jsonld-java/jsonld-java
+        //
+        //        DocumentLoader dl = new DocumentLoader();
+        //        JsonLdOptions options = new JsonLdOptions();
+        //// ... the contents of "contexts/example.jsonld"
+        //        String jsonContext = "{ \"@context\": { ... } }";
+        //        dl.addInjectedDoc("http://www.example.com/context",  jsonContext);
+        //        options.setDocumentLoader(dl);
+        //
+        //        InputStream inputStream = new FileInputStream("input.json");
+        //        Object jsonObject = JsonUtils.fromInputStream(inputStream);
+        //        Map context = new HashMap();
+        //        Object compact = JsonLdProcessor.compact(jsonObject, context, options);
+        //        System.out.println(JsonUtils.toPrettyString(compact));
```

Comment:
Do we need this comment? If it is an example a link to example might be better.

Reply:
This was the other solution.... Removed